app/src/main/kotlin/com/hyperion/ui/components/player/CommentCard.kt

```diff
@@ -32,9 +34,9 @@ fun CommentCard(
                 )
 
                 Column {
-                    Text("Comment Author")
+                    Text(stringResource(R.string.author))
                     Text(
```

Comment:
These are just temporary strings, in the future it will display the strings from the API